src/Core/Managed/PCL/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -35,5 +35,10 @@ public IDebugOutput GetDebugOutput()
             
             return this.debugOutput;
         }
+
+        public string GetEnvironmentVariable(string name)
+        {
+            return null;
```

Comment:
Is there maybe a .NET Core equivalent for getting environment variables we could use?